packages/firestore/src/api/database.ts

```diff
@@ -1925,7 +1936,8 @@ export class Query<T = firestore.DocumentData> implements firestore.Query<T> {
   constructor(
     public _query: InternalQuery,
     readonly firestore: Firestore,
-    protected readonly _converter: firestore.FirestoreDataConverter<T> | null
+    protected readonly _converter: firestore.FirestoreDataConverter<T> | null,
+    private _readTime?: SnapshotVersion
```

Comment:
`_readTime` should only be part of TargetData and not be used in the API surface.